src/pages/market/market.scss

```diff
@@ -0,0 +1,21 @@
+page-market {
+
+    ion-card{
+      text-align: center;
+      justify-content: 'center';
+      align-items: 'center'
+    }
+
+    .bottom{
+      position: relative;
+      bottom: 2px;
+    }
+
+    .orden{
+      height: 75px;
+    }
+    .topmargin{
+      margin-top:30px;
```

Comment:
Expected indentation of 4 spaces       indentation<br>Expected single space after ":"        declaration-colon-space-after